lib/jekyll/commands/clean.rb

```diff
@@ -20,17 +20,13 @@ def init_with_program(prog)
 
         def process(options)
           options = configuration_from_options(options)
-          destination = options["destination"]
-          metadata_file = File.join(options["source"], ".jekyll-metadata")
-          sass_cache = File.join(options["source"], ".sass-cache")
-
-          remove(destination, :checker_func => :directory?)
-          remove(metadata_file, :checker_func => :file?)
-          remove(sass_cache, :checker_func => :directory?)
+          remove(File.join(options["source"], ".sass-cache"))
+          remove(File.join(options["source"], ".jekyll-metadata"))
+          remove(options["destination"])
         end
 
-        def remove(filename, checker_func: :file?)
```

Comment:
*Mon ami*, is changing the signature of a public method in a minor release cycle, good practice?
Removing that parameter is going to break all those code that has passed a second argument to the `Jekyll::Commands::Clean.remove()` method..

This is best earmarked for `4.0`

Reply:
If Jekyll cared about that they wouldn't have upgraded to Liquid 4, and removed support for Liquid 3, in a breaking way... in the middle of the 3.x cycle.  Your argument has no standing.

Reply:
> Your argument has no standing.

But can you deny that this is a breaking change, either ways?

Reply:
Yes, I can deny it. Should I? Probably.  It's a command, specifically for Jekyll... its internal, if you are tapping into it, even though it's defacto private... since it's specifically for Jekyll and internal, it's not considered a semantically breaking change. Internal stuff does not fall under that category.  Only public facing API's that are meant to be publicly used.  Like Hooks.

Reply:
Lets meet half-way here and just print a deprecation message when the `arity > 1` and proceed using your refinements.. does that make sense?